### PR TITLE
(chore) O3-3964: When ordering, Stock balance doesn't display on UI(this feature was working before)

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [x]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [x] My work includes tests or is validated by existing tests.

## Summary
- Put back the quantity balance

## Related Issue
- https://openmrs.atlassian.net/browse/O3-3964

### CODE REVIEW

File: src/stock-operations/batch-no-selector/batch-no-selector.component.tsx [hunk 1]
```diff
@@ -102,7 +102,7 @@ const BatchNoSelector = <T,>(props: BatchNoSelectorProps<T>) => {
             }}
             initialSelectedItem={initialSelectedItem}
             itemToString={(s: StockBatchDTO) =>
-              s?.batchNo ? `${s?.batchNo}` : ""
+              s?.batchNo ? `${s?.batchNo} | Qty: ${s?.quantity ?? 0}` : ""
```
Comment (ojwanganto) [inline src/stock-operations/batch-no-selector/batch-no-selector.component.tsx:105]: 0 may not always be a good default. We can show a text i.e. unknown for such. thoughts?

Comment (hadijahkyampeire) [reply src/stock-operations/batch-no-selector/batch-no-selector.component.tsx:105]: okay, so I will use an empty string

File: src/stock-operations/batch-no-selector/batch-no-selector.component.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ const BatchNoSelector = <T,>(props: BatchNoSelectorProps<T>) => {
             }}
             initialSelectedItem={initialSelectedItem}
             itemToString={(s: StockBatchDTO) =>
-              s?.batchNo ? `${s?.batchNo}` : ""
+              s?.batchNo ? `${s?.batchNo} | Qty: ${s?.quantity ?? ""}` : ""
```
Comment (ojwanganto) [inline src/stock-operations/batch-no-selector/batch-no-selector.component.tsx:105]: Should we adopt some text in place of the ""? Empty string can be interpreted differently by different people and I feel we should be clear that we don't know the quantity in the batch. What do others think? @jabahum @makombe @slubwama

Comment (jabahum) [reply src/stock-operations/batch-no-selector/batch-no-selector.component.tsx:105]: I think there was a fix wea we dont even have to display the batch wen the quantity is 0. so i think just rendering the quantity would just be enough

**Comment by hadijahkyampeire** [comment]
@jabahum @ojwanganto what is the resolution on this, is it good to go in or I need to add some tweaks?

File: src/stock-operations/batch-no-selector/batch-no-selector.component.tsx (hunk 2 above)
Comment (jabahum) [reply src/stock-operations/batch-no-selector/batch-no-selector.component.tsx:105]: Can confirm to this if it still exists then move it in

Comment (hadijahkyampeire) [reply src/stock-operations/batch-no-selector/batch-no-selector.component.tsx:105]: @jabahum I don't understand what you mean

**Comment by ojwanganto** [comment]
> @jabahum @ojwanganto what is the resolution on this, is it good to go in or I need to add some tweaks?

I prefer a text like 'Unknown' instead of a blank string or a zero for the sake of the user and code readability.

**Comment by hadijahkyampeire** [comment]
@ojwanganto this should now be ready to go in, please take another look

**Review by ojwanganto — APPROVED** [review]
Thanks for this @hadijahkyampeire

**Review by jabahum — APPROVED** [review]
Thanks @hadijahkyampeire

**Pull request merged**